commands.js

```diff
@@ -2991,7 +2991,25 @@ exports.commands = {
 
 		room.game.undo(user, target);
 	},
-
+	toursexplication: 'toursguide',
+        toursguide: function (target, room, user) {
+  		if (!this.canBroadcast()) return; 
```

Comment:
You're mixing spaces & tabs, only use tabs.

Also, there is a trailing space in the last part of ``return; `` remove it.